packages/core/src/events-consumer.test.ts

```diff
@@ -73,6 +73,7 @@ describe('EventsConsumer', () => {
       await waitForNextTick();
```

Comment:
can probably revert these test changes back to main since we undid the auto advance logic